library/sinks/ChildProcess.ts

```diff
@@ -10,6 +11,17 @@ export class ChildProcess implements Wrapper {
     name: string,
     context: Context
   ): InterceptorResult {
+    // Ignore calls to spawn or spawnSync if shell option is not enabled
+    if (name === "spawn" || name === "spawnSync") {
+      const shellOption = args.find(
+        (arg) => isPlainObject(arg) && "shell" in arg
+      );
+
+      if (!shellOption) {
+        return undefined;
+      }
```

Comment:
If `shell` is `false`, then it will still continue to the checks below?

![Screenshot 2024-04-09 at 17 45 13](https://github.com/AikidoSec/runtime-node/assets/3886384/4711d39e-f1fa-4087-b9cd-e586136f6583)

`"shell" in arg` will return true if the property is available on the object but it doesn't check its value?